pkg/build/graph/analysis/bc.go

```diff
@@ -267,16 +280,34 @@ func validImageStreamImage(imageNode *imagegraph.ImageStreamImageNode, imageStre
 		compareTime := checkTime.Add(5 * time.Minute)
 		currentTime, _ := time.Parse(time.RFC3339, unversioned.Now().UTC().Format(time.RFC3339))
 		if compareTime.Before(currentTime) {
-			return false, dockerImageReference.ID, "import the image with hexadecimal ID %s into the image stream %s"
+			return osgraph.Suggestion(fmt.Sprintf("`oc import-image %s --from=` where `--from` specifies an image with hexadecimal ID %s", imageStream.GetName(), imageID))
 		}
 
-		return false, dockerImageReference.ID, "a import of the image with hexadecimal ID %s into the image stream %s could be in progress; check again after a couple of minutes"
+		return osgraph.Suggestion(fmt.Sprintf("`oc import-image %s --from=` with hexadecimal ID %s possibly in progress", imageStream.GetName(), imageID))
+
+	}
+	return osgraph.Suggestion(fmt.Sprintf("Possible error occurred with `oc import-image %s --from=` with hexadecimal ID %s; inspect images stream annotations", imageStream.GetName(), imageID))
```

Comment:
I would love to have https://github.com/kubernetes/kubernetes/issues/19817 in place for this. Then we could have some thing `" 'oc annotate is/%s' to inspect for possible import error"`